docs/Select-Canonical-Transcript.md

```diff
@@ -0,0 +1,106 @@
+- `mskcc`
+- `uniprot` (default)
+- `genome_nexus`
+- `ensembl`
+
+The list of canonical transcript isoform override for each gene can be found here, which is genereated by this [script](https://github.com/genome-nexus/genome-nexus-importer/blob/master/scripts/make_one_canonical_transcript_per_gene.py): 
+- GRCh37: [canonical_transcripts_per_hgnc.txt](https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/grch37_ensembl92/export/ensembl_biomart_canonical_transcripts_per_hgnc.txt)
+- GRCh38: 
+  - [canonical_transcripts_per_hgnc.txt (ensembl95)](https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/grch38_ensembl95/export/ensembl_biomart_canonical_transcripts_per_hgnc.txt)
+  - [canonical_transcripts_per_hgnc.txt (ensembl92)](https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/grch38_ensembl92/export/ensembl_biomart_canonical_transcripts_per_hgnc.txt)
+
+
+The isoform override files can be found here:
+- `mskcc`:
+    - GRCh37: 
+      - [isoform_overrides_oncokb_grch37.txt](https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/common_input/isoform_overrides_oncokb_grch37.txt)
+        - Source: [OncoKB curated genes](https://www.oncokb.org/api/v1/utils/allCuratedGenes.txt?includeEvidence=false)
+        - Description: This file contains the GRCh37 OncoKB isoform canonical transcript list. It is the primary choice when `mskcc` is selected as the isoform override source.
+      - [isoform_overrides_at_mskcc_grch37.txt](https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/common_input/isoform_overrides_at_mskcc_grch37.txt)
+        - Source: [MSKCC VCF2MAF Isoform Overrides](https://github.com/mskcc/vcf2maf/blob/main/data/isoform_overrides_at_mskcc)
+        - Description: This file contains the GRCh37 mskcc isoform canonical transcript list. If a transcript is not present in the OncoKB canonical transcript list, this file is used as a fallback.
+
+    - GRCh38: 
+      - [isoform_overrides_oncokb_grch38.txt](https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/common_input/isoform_overrides_at_mskcc_grch38.txt)
+        - Source: [OncoKB curated genes](https://www.oncokb.org/api/v1/utils/allCuratedGenes.txt?includeEvidence=false)
+        - Description: This file contains the GRCh38 OncoKB isoform canonical transcript list. It is the primary choice when `mskcc` is selected as the isoform override source.
+      - [isoform_overrides_at_mskcc_grch38.txt](https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/common_input/isoform_overrides_genome_nexus_grch38.txt)
+        - Source: [MSKCC VCF2MAF Isoform Overrides](https://github.com/mskcc/vcf2maf/blob/main/data/isoform_overrides_at_mskcc_grch38)
+        - Description: This file contains the GRCh38 MSKCC isoform canonical transcript list. If a transcript is not present in the OncoKB canonical transcript list, this file is used as a fallback.
+
+- `uniprot`:
```

Comment:
I think it's good to mention the caveats around `uniprot` as outlined here: https://docs.cbioportal.org/mutation-data-transcript-annotation/#transcript-assignment. The name `uniprot` seems to imply that it comes from uniprot somehow so can be confusing to a user